src/main/java/it/gov/pagopa/pu/debtpositions/service/create/receipt/CreateReceiptServiceImpl.java

```diff
@@ -40,8 +41,9 @@ public ReceiptDTO createReceipt(ReceiptWithAdditionalNodeDataDTO receiptDTO, Str
 
     //check if the same receipt is already present on DB
     Optional<ReceiptDTO> receiptInDb = checkIfAlreadyStored(receiptDTO);
-    if (receiptInDb.isPresent())
+    if (receiptInDb.isPresent() && StringUtils.isBlank(receiptDTO.getIud())) {
```

Comment:
let's describe why we are performing this
```suggestion
    // if the receipt already exists, we should continue only if the input IUD exists, in order to check the existence of a DP having an Installment with the requested IUD: if it doesn't exists, we should create a new technical DP with the requested Installment (use case heterogeneous IUV: many DP on same org having same IUV)
    if (receiptInDb.isPresent() && StringUtils.isBlank(receiptDTO.getIud())) {
```